app/views/contact_methods/_whatsapp.html.erb

```diff
@@ -0,0 +1,32 @@
+<div class="contact-panel contact-panel__chat-open" id="js-whatsapp">
+  <h2 aria-level="2" class="contact-panel__heading" role="heading">
+    <%= render 'contact_methods/whatsapp_logo' %>WhatsApp
+  </h2>
+  <p class="contact-panel__chat-javascript">
+    <%= t('contact_panels.webchat.nojs_html') %></a>.
+  </p>
+  <div class="contact-panel__dynamic-content is-loaded" id="js-whatsapp-content">
+    <ul class="contact-panel__chat-description contact-panel__list unstyled-list" id="js-whatsapp-description">
```

Comment:
This content is not being displayed at all at smaller widths (<480px). Not sure why.

Reply:
This was coming from Frontend which right now is hardcoded so new styles are in place to override it for now